c/CMakeLists.txt

```diff
@@ -33,5 +33,50 @@ cmake_minimum_required(VERSION 3.10)
 # IMPORTED option, which only defines the targets within the scope of the library.
 # This allows the same target name to be used for different NVTX versions.
 
+# Project declaration needs to be before the configuration
+project(nvtx3 VERSION 3.1.1)
+
+# Add installation rules
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
 set(NVTX3_TARGETS_NOT_USING_IMPORTED ON)
 include(nvtxImportedTargets.cmake)
+
```

Comment:
You need to add `set_target_properties(nvtx3-c PROPERTIES EXPORT_PROPERTIES VERSION)` after the include so that we match the version property checks in the `nvtxImportedTargets.cmake`

Reply:
I added this for `nvtx3-c` and also `nvtx3-cpp` -- does it look right now? I am not 100% sure on this.

Reply:
Looks great